.github/workflows/upload-and-deploy.yaml

```diff
@@ -133,10 +133,12 @@ jobs:
         run: |
           pipenv install ecs_logging~=2.0
 
-      - name: Test lambda scripts with pytest
+      - name: Test scripts with pytest (lambda, etc.)
         run: |
           pipenv run python -m pytest tests/test_s3_event_config_lambda.py -v
           pipenv run python -m pytest tests/test_s3_to_glue_lambda.py -v
+          pipenv run python -m pytest tests/test_lambda_dispatch.py -v
+          pipenv run python -m pytest tests/test_consume_logs.py -v
```

Comment:
Should these each be their own separate steps?

Reply:
Actually they can all be grouped under one command this way they are run all at once, and if one set of tests have failures, the other tests in another test suite still get run.